website/docs/d/fabric_network.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "vra"
+page_title: "VMware vRealize Automation: fabric_network"
+description: |-
+  Provides a data lookup for vRA fabric networks.
+---
+
+# Data Source: fabric_network
+## Example Usages
+This is an example of how to lookup fabric networks.
+
+**Fabric network data source by Id:**
+
+```hcl
+# Lookup vRA fabric network using its Id
+data "fabric_network" "this" {
+  id = var.fabric_network_id
+}
+```
+
+**Fabric network by filter query:**
+
+```hcl
+# Lookup vRA fabric network using its name
+data "fabric_network" "this" {
+  filter = "name eq '${var.name}'"
+}
+```
+
+A fabric network data source supports the following arguments:
+
+## Argument Reference
+* `filter` - Filter query string that is supported by vRA multi-cloud IaaS API. Only one of 'filter' or 'id' must be specified.
+
+* `id` - The id of the vRA fabric network.  Only one of 'filter' or 'id' must be specified.
```

Comment:
`id` is not supported. Please remove it and update the `filter` to mark it as `Required` and remove

> Only one of 'filter' or 'id' must be specified.